data/src/main/java/io/gnosis/data/models/TransferDetails.kt

```diff
@@ -44,17 +48,19 @@ data class SettingsChangeDetails(
     val txStatus: TransactionStatus,
     val createdAt: Date?,
     val executedAt: Date?,
+    val executor: Solidity.Address,
     val detailedExecutionInfo: DetailedExecutionInfo?,
     val txData: TxData? // contains data decoded
-)
+) : TransactionDetails
 
 // Safe created
-data class CreationChangeDetails(
+data class CreationDetails(
```

Comment:
I would display the creation details in the same screen/ models as the other transactions, as this is a very special transaction with a compeltely different design ... also this information will never be returned by the gateway.